hw/ip/hmac/dv/cov/hmac_cov_excl.el

```diff
@@ -18,273 +18,169 @@ CHECKSUM: "3109872245 3727953184"
-Condition 4 "1115643068" "(gen_normal_fifo.fifo_empty && wvalid) 1 -1" (1 "01")
-Condition 4 "1115643068" "(gen_normal_fifo.fifo_empty && wvalid) 1 -1" (3 "11")
-Condition 2 "3256391329" "((gen_normal_fifo.wptr_msb == gen_normal_fifo.rptr_msb) ? ((1'(gen_normal_fifo.wptr_value) - 1'(gen_normal_fifo.rptr_value))) : (((1'(Depth) - 1'(gen_normal_fifo.rptr_value)) + 1'(gen_normal_fifo.wptr_value)))) 1 -1" (1 "0")
 Condition 3 "3985904135" "((gen_normal_fifo.fifo_empty && wvalid) ? wdata : gen_normal_fifo.storage_rdata) 1 -1" (2 "1")
-CHECKSUM: "2951258608 3264379853"
-INSTANCE: tb.dut
-ANNOTATION: "[UNR]"
-Condition 3 "1718103215" "(hmac_fifo_wsel && fifo_wready) 1 -1" (2 "10")
-Condition 1 "157377054" "(reg_fifo_wvalid && fifo_wready && ((!hmac_fifo_wsel))) 1 -1" (3 "110")
-CHECKSUM: "2182036441 4271210983"
+Condition 2 "3256391329" "((gen_normal_fifo.wptr_msb == gen_normal_fifo.rptr_msb) ? ((1'(gen_normal_fifo.wptr_value) - 1'(gen_normal_fifo.rptr_value))) : (((1'(Depth) - 1'(gen_normal_fifo.rptr_value)) + 1'(gen_normal_fifo.wptr_value)))) 1 -1" (1 "0")
+Condition 4 "1115643068" "(gen_normal_fifo.fifo_empty && wvalid) 1 -1" (3 "11")
+Condition 4 "1115643068" "(gen_normal_fifo.fifo_empty && wvalid) 1 -1" (1 "01")
+CHECKSUM: "2768112785 1802779848"
+INSTANCE: tb.dut.u_tlul_adapter
+ANNOTATION: "[UNSUPPORTED] hmac msg fifo is WO"
+Condition 7 "4228513916" "((((reqfifo_rdata.op == OpRead) & (~reqfifo_rdata.error))) ? reqfifo_rready : 1'b0) 1 -1" (2 "1")
+CHECKSUM: "3518855206 1814878003"
+INSTANCE: tb.dut.u_packer
+Condition 2 "2611056762" "(flush_i && ready_i) 1 -1" (2 "10")
+CHECKSUM: "3885231045 4271210983"
 INSTANCE: tb.dut.u_reg.u_socket.err_resp
-ANNOTATION: "[UNR] hmac msg fifo is WO, and error resp is handled at tlul interface level"
-Branch 0 "2051317344" "(err_opcode == Get)" (1) "(err_opcode == Get) 0"
+ANNOTATION: "[UNR] error resp is handled at tlul interface level"
 Branch 2 "2710626646" "(!rst_ni)" (1) "(!rst_ni) 0,1"
-Branch 1 "2688379704" "(!rst_ni)" (1) "(!rst_ni) 0,1,-"
+ANNOTATION: "[UNSUPPORTED] hmac msg fifo is WO"
+Branch 0 "2051317344" "(err_opcode == Get)" (1) "(err_opcode == Get) 0"
 Branch 1 "2688379704" "(!rst_ni)" (3) "(!rst_ni) 0,0,0"
+ANNOTATION: "[UNR] hmac msg fifo is WO, and error resp is handled at tlul interface level"
+Branch 1 "2688379704" "(!rst_ni)" (1) "(!rst_ni) 0,1,-"
 CHECKSUM: "2494051891 720836556"
 INSTANCE: tb.dut.u_sha2
 ANNOTATION: "[UNR] FUTURE ITEM: throughput optimization"
 Branch 7 "2566632963" "fifo_st_q" (2) "fifo_st_q FifoLoadFromFifo ,-,1,-,-,-,-"
 Branch 9 "1372145373" "sha_st_q" (6) "sha_st_q ShaUpdateDigest ,-,-,-,1"
-CHECKSUM: "2443667993 1561326489"
+ANNOTATION: "[UNSUPPORTED] FUTURE ITEM: wipe secret"
+Branch 0 "382698320" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-,-,-"
+Branch 1 "3205957143" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-"
+Branch 2 "1214035098" "(!rst_ni)" (1) "(!rst_ni) 0,1,-,-,-"
```

Comment:
This is what I missed in the previous exclusion. In the previous version, I tried wipe secret functionality and accidentally added it to the coverage collection. Even though I manually add the wipe secret back, but I missed these three.